pages/learn/manage/billing.md

```diff
@@ -14,53 +15,75 @@ From this dashboard, you can change your billing plan, update your billing infor
+For both devices and users, usage is calculated based on the maximum number in your account at any point during the month after any available credits have been used. **This includes all [active devices][inactive], whether online or offline.**
 
 As an example, let's say you are on the Pilot plan, which includes up to 50 devices and 3 users. In one month, your active device count steadily climbs from 43 to 52. During this same month, you temporarily increase your user count from 3 to 4, before returning it to 3 at the end of the month. Based on this usage, your dynamic billing charges for the month will include an additional 2 devices and 1 user.
 
 Furthermore, we charge a device or user only once for the most feature-rich usage tier within the same billing period. For example, a user was assigned a Developer role for the first half of the billing period and was later reassigned the Operator role for the second half of the billing period. In this scenario, we would charge the usage as a Developer role for the entire billing period. This is because the Developer role is a more feature-rich usage tier than the Operator role, as seen in [fleet membership types][fleet-membership-types]. It's important to note that we are not double charging for the same user.
 
-Charges for usage outside your plan will show up under the *Add-ons* section of your invoice. If you prefer, you can still contact us to pre-purchase devices and users and have them added to your plan, rather than be automatically billed in arrears for additional devices and users.
+Charges for usage outside your plan will show up under the _Add-ons_ section of your invoice. If you prefer, you can still contact us to pre-purchase devices and users and have them added to your plan, rather than be automatically billed in arrears for additional devices and users.
 
 ## Inactive devices
 
 If you have devices that will be inactive for long periods of time, you can mark them as `Inactive`. This will keep them from being counted towards your device total. We’ll charge a one-time deactivation fee, and the devices will remain inactive until they come online.
 
-Marking a device as `Inactive` is a [device action][device-action]. Device actions can be applied in one of three ways: as a group action applied to one or more devices in the device list, as an action selected from the dropdown on the device dashboard, or directly from the *Actions* tab of the device dashboard. Once a device has been marked `Inactive`, it will be faded out in the device list.
+Marking a device as `Inactive` is a [device action][device-action]. Device actions can be applied in one of three ways: as a group action applied to one or more devices in the device list, as an action selected from the dropdown on the device dashboard, or directly from the _Actions_ tab of the device dashboard. Once a device has been marked `Inactive`, it will be faded out in the device list.
```

Comment:
While you're here, it's not actually under the actions tab anymore and it's not considered an action. It is a `device setting` (not sure what to do with the `device-action` bit, maybe that stays the same?) under the `Settings` tab of the device dashboard

Reply:
@myarmolinsky Good find. I think I will include this update when the whole page is audited (I plan to do this soon)

Reply:
done